### PR TITLE
v2.2.0 release checklist

### PR DESCRIPTION
**Pull request by keeramis**
### Description

Release checklist for 2.2.0 LTS backporting.

### List of changes

- Do not reset the DTLS session on socket errors [#2335](https://github.com/particle-iot/device-os/pull/2335)
- On-device tests to validate [#2335](https://github.com/particle-iot/device-os/pull/2335) [#2337](https://github.com/particle-iot/device-os/pull/2337)
- Fix warm boot test process for on-device tests [72177c85](https://github.com/particle-iot/device-os/pull/2341/commits/72177c855d55949bfbee0fb990d3605e9dc46001)

### Others

- Fixes bootloader dependency error in 2.2.0-rc.1 [ch82369](https://app.clubhouse.io/particle/story/82369/fix-bootloader-dependency-for-2-2-0-rc-1)

### References

---

### Completeness

- [x] User is totes amazing for contributing!
- [ ] Contributor has signed CLA ([Info here](https://github.com/spark/firmware/blob/develop/CONTRIBUTING.md))
- [ ] Problem and Solution clearly stated
- [ ] Run unit/integration/application tests on device
- [ ] Added documentation
- [x] Added to CHANGELOG.md after merging (add links to docs and issues)

### CODE REVIEW

**Comment by keeramis** [comment]
(Still to fix the photon/p1 overflow.)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## 2.2.0
+
+### BUGFIXES
+
```
Comment (avtolstoy) [inline CHANGELOG.md:4]: This needs to be the full changelog including everything contained within 2.2.0-rcs

File: system/system-versions.md [hunk 2]
```diff
@@ -92,7 +92,7 @@
 | 300 | 1200 | 1.2.0-alpha.1 |       Photon, P1, Electron, Boron |
 | 300 | 1201 | 1.2.0-beta.1  |       Core, Photon, P1, Electron, Xenon, Argon, Boron |
 | 302 | 1202 | 1.2.0-rc.1    |       Core, Photon, P1, Electron, Xenon, Argon, Boron, X SoM, A SoM, B SoM |
-| 310 | 1210 | 1.2.1-rc.1    |       Core, Photon, P1, Electron, Xenon, Argon, Boron, X SoM, A SoM, B SoM |
+| 310 | 1210 | 1.2.1-rc.1    |       Core, Photon, P1, Electron, Xenon, Argon, Boron, X SoM, A SoM, B SoM |w
```
Comment (avtolstoy) [inline system/system-versions.md:95]: Extra `w`

**Review by avtolstoy — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (keeramis) [reply CHANGELOG.md:15]: Fixed. And mentioning #2335 as `[Cellular] Fix high data usage during poor signal conditions [#2335](https://github.com/particle-iot/device-os/pull/2335)`

File: system/system-versions.md (hunk 2 above)
Comment (keeramis) [reply system/system-versions.md]: Fixed

**Pull request closed without merging**